megamek/src/megamek/common/templates/VehicleTROView.java

```diff
@@ -44,6 +45,13 @@ public VehicleTROView(Tank tank) {
 
     @Override
     protected String getTemplateFileName(boolean html) {
+        if(tank instanceof GunEmplacement) {
+            if (html) {
+                return "gunemplacement.ftlh";
+            }
+            return "gunemplacement.ftl";
```

Comment:
```suggestion
            if (html) {
                return "gunemplacement.ftlh";
            } else {
                return "gunemplacement.ftl";
            }
```

Reply:
I find it easier to read if the return is in an else in statements like this, but this is very much a suggestion.